packages/site/src/pages/index.tsx

```diff
@@ -125,12 +126,12 @@ const Index = () => {
       // The array may contain more than one account in the future. 
       // The first account in the array will always be considered the user's "selected" account.
       // https://docs.metamask.io/wallet/get-started/access-accounts
-      // TODO address the fact that this "selected" account is different from the one the snap uses to send the tx
-      await sendTransaction(
-        // accounts[0],
-        '0x3a404155a41cd28578041d041e7de86941ceaab7',
-        ethers.utils.parseUnits("1", "wei").toString()
-      );
+      // TODO address the fact that this "se
+      const tx: CeloTransactionRequest = {
+        to: '0x3a404155a41cd28578041d041e7de86941ceaab7',
+        value: Number(ethers.utils.parseUnits("1", "wei"))
```

Comment:
Why are you attempting to cast this to a number here? The type returned by parseUnits is already compatible

Reply:
For some weird reasons, sending a transaction with something like 1 Celo does not work. fails with below error  

but it works for small numbers like e.g 0.1 Celo

```
bundle.js:6260 Error: JSON-RPC responses must be JSON serializable objects.
  at IFrameSnapExecutor.respond (bundle.js:5545:4285)
  at IFrameSnapExecutor.onCommandRequest (bundle.js:5545:3937)
```